### PR TITLE
LINK-1930 | Talpa product mapping and accounting

### PR DESCRIPTION
**Pull request by harriris-vincit**
### Description
Adds business logic for creating a "product mapping" and a related "accounting" instances in the Talpa API when a `registration` is being created or updated and it is a registration with price group selections.
### Closes
[LINK-1930](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1930)

[LINK-1930]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1930?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr883.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr883.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr883.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr883.api.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**LINKEDEVENTS-API branch is deployed to platta: https://linkedevents-pr883.api.dev.hel.ninja** :rocket::rocket::rocket:

File: registrations/models.py [hunk 1]
```diff
@@ -1623,9 +1719,20 @@ class SignUpPriceGroup(RegistrationPriceGroupBaseModel, SoftDeletableBaseModel):
     description = models.CharField(max_length=255)
 
     def to_web_store_order_json(self):
-        # TODO: use a non-hardcoded productId once product mapping is implemented.
```
Comment (nicobav) [inline registrations/models.py:1626]: 🏅

**Review by nicobav — APPROVED** [review]

**Pull request merged**